cryptkeeper.go

```diff
@@ -146,15 +147,25 @@ func (cb CryptBytes) Value() (value driver.Value, err error) {
 func SetCryptKey(secretKey []byte) error {
 	keyLen := len(secretKey)
 	if keyLen != 16 && keyLen != 24 && keyLen != 32 {
-		return fmt.Errorf("Invalid KEY to set for CRYPT_KEEPER_KEY; must be 16, 24, or 32 bytes (got %d)", keyLen)
+		var err error
+		if secretKey, err = pkcs7Pad(secretKey); err != nil {
+			return err
+		}
 	}
 	cryptKeeperKey = secretKey
 	return nil
 }
 
 // CryptKey will return the current key that will be used for encryption and decryption
 func CryptKey() []byte {
-	return cryptKeeperKey
+	if cryptKeeperKey == nil {
+		return cryptKeeperKey
+	}
+	key, err := pkcs7Unpad(cryptKeeperKey)
+	if err != nil {
+		panic("bug")
```

Comment:
can you put a more descriptive message here like 
```suggestion
		panic(fmt.Errorf("unpad of key failed, this should not happen: %s", err))
```